app/enterprise/2.3.x/admin-api/licenses/examples.md

```diff
@@ -71,10 +79,14 @@ http GET :8001/licenses/30b4edb7-0847-4f65-af90-efbed8b0161f
 
 ### Auto-generated ID
 
+To generate an ID automatically, submit a `POST` request directly to `/licenses`:
+
 ```bash
-http POST :8001/licenses payload="{\"license\":{\"payload\":{\"admin_seats\":\"1\",\"customer\":\"Example Company, Inc\",\"dataplanes\":\"1\",\"license_creation_date\":\"2017-07-20\",\"license_expiration_date\":\"2017-07-20\",\"license_key\":\"00141000017ODj3AAG_a1V41000004wT0OEAU\",\"product_subscription\":\"Konnect Enterprise\",\"support_plan\":\"None\"},\"signature\":\"6985968131533a967fcc721244a979948b1066967f1e9cd65dbd8eeabe060fc32d894a2945f5e4a03c1cd2198c74e058ac63d28b045c2f1fcec95877bd790e1b\",\"version\":\"1\"}}"
+$ http POST :8001/licenses \
+  payload='{\"license\":{\"payload\":{\"admin_seats\":\"1\",\"customer\":\"Example Company, Inc\",\"dataplanes\":\"1\",\"license_creation_date\":\"2017-07-20\",\"license_expiration_date\":\"2017-07-20\",\"license_key\":\"00141000017ODj3AAG_a1V41000004wT0OEAU\",\"product_subscription\":\"Konnect Enterprise\",\"support_plan\":\"None\"},\"signature\":\"6985968131533a967fcc721244a979948b1066967f1e9cd65dbd8eeabe060fc32d894a2945f5e4a03c1cd2198c74e058ac63d28b045c2f1fcec95877bd790e1b\",\"version\":\"1\"}}'
```

Comment:
@lena-larionova The change doesn't appear to work locally for me; however the original one does:

#### Original
```
[kong-ee-dc0112767:/kong]# http POST :8001/licenses payload="{\"license\":{\"payload\":{\"admin_seats\":\"1\",\"customer\":\"Example Company, Inc\",\"dataplanes\":\"1\",\"license_creation_date\":\"2017-07-20\",\"license_expiration_date\":\"2017-07-20\",\"license_key\":\"00141000017ODj3AAG_a1V41000004wT0OEAU\",\"product_subscription\":\"Konnect Enterprise\",\"support_plan\":\"None\"},\"signature\":\"6985968131533a967fcc721244a979948b1066967f1e9cd65dbd8eeabe060fc32d894a2945f5e4a03c1cd2198c74e058ac63d28b045c2f1fcec95877bd790e1b\",\"version\":\"1\"}}"
HTTP/1.1 201 Created
Access-Control-Allow-Origin: *
Connection: keep-alive
Content-Length: 597
Content-Type: application/json; charset=utf-8
Date: Mon, 22 Feb 2021 21:14:31 GMT
Server: kong/2.3.2.0-dev-enterprise-edition
X-Kong-Admin-Latency: 12
X-Kong-Admin-Request-ID: kr8iin0Y14QZTKUZoudmx4AJFFL67WMb
vary: Origin

{
    "created_at": 1614028471,
    "id": "c1d0eb24-2f78-4e45-ad36-f88aac73496a",
    "payload": "{\"license\":{\"payload\":{\"admin_seats\":\"1\",\"customer\":\"Example Company, Inc\",\"dataplanes\":\"1\",\"license_creation_date\":\"2017-07-20\",\"license_expiration_date\":\"2017-07-20\",\"license_key\":\"00141000017ODj3AAG_a1V41000004wT0OEAU\",\"product_subscription\":\"Konnect Enterprise\",\"support_plan\":\"None\"},\"signature\":\"6985968131533a967fcc721244a979948b1066967f1e9cd65dbd8eeabe060fc32d894a2945f5e4a03c1cd2198c74e058ac63d28b045c2f1fcec95877bd790e1b\",\"version\":\"1\"}}",
    "updated_at": 1614028471
}
```

#### Changed; quotes removed

```
[kong-ee-dc0112767:/kong]# http POST :8001/licenses \
>   payload='{\"license\":{\"payload\":{\"admin_seats\":\"1\",\"customer\":\"Example Company, Inc\",\"dataplanes\":\"1\",\"license_creation_date\":\"2017-07-20\",\"license_expiration_date\":\"2017-07-20\",\"license_key\":\"00141000017ODj3AAG_a1V41000004wT0OEAU\",\"product_subscription\":\"Konnect Enterprise\",\"support_plan\":\"None\"},\"signature\":\"6985968131533a967fcc721244a979948b1066967f1e9cd65dbd8eeabe060fc32d894a2945f5e4a03c1cd2198c74e058ac63d28b045c2f1fcec95877bd790e1b\",\"version\":\"1\"}}'
HTTP/1.1 400 Bad Request
Access-Control-Allow-Origin: *
Connection: keep-alive
Content-Length: 215
Content-Type: application/json; charset=utf-8
Date: Mon, 22 Feb 2021 21:14:41 GMT
Server: kong/2.3.2.0-dev-enterprise-edition
X-Kong-Admin-Latency: 3
X-Kong-Admin-Request-ID: 39sC7lcMEna7fqz5NnriAogGWZi6BG57
vary: Origin

{
    "code": 2,
    "fields": {
        "payload": "Unable to validate license: could not decode license json"
    },
    "message": "schema violation (payload: Unable to validate license: could not decode license json)",
    "name": "schema violation"
}
```

If you want to remove the starting quotes the escaping of the double quote will also need to be removed:

```
http :8001/licenses payload='{"license":{"payload":{"admin_seats":"1","customer":"Example Company, Inc","dataplanes":"1","license_creation_date":"2017-07-20","license_expiration_date":"2017-07-20","license_key":"00141000017ODj3AAG_a1V41000004wT0OEAU","product_subscription":"Konnect Enterprise","support_plan":"None"},"signature":"6985968131533a967fcc721244a979948b1066967f1e9cd65dbd8eeabe060fc32d894a2945f5e4a03c1cd2198c74e058ac63d28b045c2f1fcec95877bd790e1b","version":"1"}}'
```

Reply:
And this is why I wanted to come back and test it... eek. Thanks!

Reply:
Ideally, the less work for the user, the better - so if they don't have to escape every double quote, this is definitely the preferable option.